lighthouse-core/lib/page-functions.js

```diff
@@ -98,6 +98,10 @@ function getElementsInDocument(selector) {
       if (el.shadowRoot) {
         _findAllElements(el.shadowRoot.querySelectorAll('*'));
       }
+      // If the element has a contentDocument (IFrame), dig deeper.
```

Comment:
only some iframes have contentDocuments... I honestly dont understand why some do and some dont (even within google ads iframes...)
![image](https://user-images.githubusercontent.com/39191/62501592-647bdc00-b7a0-11e9-857b-9ef8ad9958d3.png)

1) can we make this comment more specific to which iframes are included
2) i think we need this behind a `pierceIframes` option that defaults to off, otherwise our `dom-size` audit is about to go pretty crazy.